src/store/models/network.spec.ts

```diff
@@ -495,6 +504,53 @@ describe('Network model', () => {
     });
   });
 
+  describe('Updating Taro Backend', () => {
+    beforeEach(async () => {
+      createTaroNetwork();
+    });
```

Comment:
Place these tests in the `describe("Taro network")` group to eliminate the need for the `createTaroNetwork()` func.